{"uuid":"b3a9dbf1-4a8d-44af-a101-e2366ab7b245","name":"Scenario Outline: The user is presented with error message for invalid code in tryEditor for Max Consecutive Ones","children":["ed531b4a-8a66-4800-9eff-fd2abf95a9da"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696720446608,"stop":1696720446608},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696720446690,"stop":1696720446691},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696720446745,"stop":1696720446745},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696720448767,"stop":1696720448767}],"start":1696720446254,"stop":1696720448767}